### PR TITLE
post: happy new year

### PR DESCRIPTION
**Pull request by cexbrayat**
Short but I'm lacking inspiration, feel free to improve.
When we're happy with this, I'll write the mailing.

### CODE REVIEW

File: _posts/2016-01-08-new-angular-2-chapter-for-new-year.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+description: "Angular 2 is out in beta, our ebook is a few days shy from being released, but here are some new free chapters!"
+---
+
+# Happy new year!
+
+We hope you enjoyed your Christmas break.
+Maybe you haven’t heard but Angular 2 is now out in beta!
+That’s not the final release yet, but we’re getting there.
+And we have been hard at work on [our ebook](https://books.ninja-squad.com/angular2).
+**In fact we are just a few days shy from our first release.**
```
Comment (clacote) [inline _posts/2016-01-08-new-angular-2-chapter-for-new-year.md]: I've never heard that saying "a few days shy", but that's might be my lack of English skills.

File: _posts/2016-01-08-new-angular-2-chapter-for-new-year.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Like the framework we are going to release a first version of the ebook
+(in english and in french) with all
```
Comment (clacote) [inline _posts/2016-01-08-new-angular-2-chapter-for-new-year.md]: English and in French

File: _posts/2016-01-08-new-angular-2-chapter-for-new-year.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+that you need to write your first app,
+and then we’ll release some updates with new chapters on more advanced topics.
+
+To keep you busy while you wait for the ebook release,
+we added some content to the free samples.
+
+You can now read about :
```
Comment (clacote) [inline _posts/2016-01-08-new-angular-2-chapter-for-new-year.md]: No space before ":"

**Comment by clacote** [comment]
I'll fix the typos in a PR I'm gonna try to push tonight, with French version, more content and some pictures to have a more fancy blog post and mailing draft.

**Comment by clacote** [comment]
It has all been proposed in #165.

**Comment by clacote** [comment]
Also, you should maybe add something about the router state: it's buggy in Angular and will be rewritten, and our chapter will wait for it.

File: _posts/2016-01-08-new-angular-2-chapter-for-new-year.md (hunk 1 above)
Comment (jnizet) [inline _posts/2016-01-08-new-angular-2-chapter-for-new-year.md]: Same here. It's in the description too.

**Comment by clacote** [comment]
@cedric: do you want me to add a sentence about router state, or do you think it is not useful?

**Comment by cedric** [comment]
@clacote Wrong Cedric I think...

**Comment by cexbrayat** [comment]
No that's OK

**Comment by clacote** [comment]
@cedric : oh yes, sorry about that (I was writing this on a mobile without autocompletion on mention)...

**Pull request merged**